website_sale_wishlist/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+=================
+Products Wishlist
+=================
+
+This module extends the functionality of your eCommerce to support a wishlist
+of products.
+
+Configuration
+=============
+
+This module includes some templates, enabled by default. To disable them:
+
+#. Go to `your shop </shop>`_.
+#. Click on *Customize*.
+#. Uncheck *Add to Wishlist From List*.
+#. Enter any product.
+#. Click on *Customize*.
+#. Uncheck *Add to Wishlist From Form*.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to `your shop </shop>`_.
+#. Press on the heart icon for any product.
+#. You have now the top *My Wishlist* menu.
```

Comment:
`#. Clicking on it, you go to a filtered view of your shop with only the wishlisted elements on it`